lib/iris/unit.py

```diff
@@ -1823,9 +1823,9 @@ def convert(self, value, other, ctype=FLOAT64):
             >>> import numpy as np
             >>> c = unit.Unit('deg_c')
             >>> f = unit.Unit('deg_f')
-            >>> print(c.convert(0, f))
+            >>> print(np.round(c.convert(0, f), 8))
```

Comment:
Actually, Python 2 and 3 return the same result; it's just not 32, it's 31.999999999999886.